### PR TITLE
VUE-557 Kiva Classic Typography / Tailwind branch into dev 

### PR DESCRIPTION
**Pull request by ryan-ludwig**
* Removes the need for `.kv-tailwind` class, and makes Tailwind available on all UI pages. 
* Updates typography styles to match Kiva Classic across UI
* Updates form elements to use kv-components
* Update colors to match Kiva Classic across UI
* Misc updates to swap out components with new kv-components

### CODE REVIEW

File: src/components/15Years/15YearsHowKivaWorks.vue [hunk 1]
```diff
@@ -83,16 +83,11 @@ export default {
 /* stylelint-disable no-descending-specificity */
 
 .works {
-	background: #fff;
-	color: $twilight;
-
 	/* hides the horizontal scrollbar created by hacking the carousel below */
 	width: 100%;
 	overflow-x: hidden;
```
Comment (mcstover) [inline src/components/15Years/15YearsHowKivaWorks.vue:88]: Seems like these are prob non operational now that there isn't a "root" instance of the "works" class... prob doesn't matter too much though for this component.

**Review by emuvente — APPROVED** [review]
A lot of files to look over, but it all looks good to me!

**Review by mcstover — APPROVED** [review]
Dive in!

**Comment by ryan-ludwig** [comment]
All of these changes have been reviewed once before when going into `typography_dev`, but wanted to make sure people could see what was coming.

**Review by roger-in-kiva — APPROVED** [review]
Nice work!

File: src/assets/icons/inline/pencil.svg [hunk 2]
```diff
@@ -1,3 +1,3 @@
 <svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
-    <path d="M15.405 0L1.74 13.666l4.594 4.594L20 4.595 15.405 0zM1.208 14.67L0 20l5.33-1.208-4.122-4.121z" fill="#118aec"/>
+    <path d="M15.405 0L1.74 13.666l4.594 4.594L20 4.595 15.405 0zM1.208 14.67L0 20l5.33-1.208-4.122-4.121z" class="tw-fill-current"/>
```
[the remark this answers is not in the document]
Comment (ryan-ludwig) [reply src/assets/icons/inline/pencil.svg:2]: This lets the parent set the color using text-color
Something like
`<pencil class="tw-text-secondary" />`

**Review by BoulderBrains — APPROVED** [review]

**Review by eddieferrer — APPROVED** [review]

**Pull request merged**